src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Update GenevaTraceExporter to export `activity.TraceStateString` as the value
```

Comment:
nit: 
1. make it clear that this is opt-in feature. i.e not enabled by default.
2. make it clearer that this is only for Spans.

Reply:
I've updated the CHANGELOG and also added a section about this option in the README file.